Formula/dhall.rb

```diff
@@ -16,10 +17,11 @@ class Dhall < Formula
   end
 
   depends_on "cabal-install" => :build
-  depends_on "ghc@8.6" => :build
+  depends_on "ghc" => :build
 
   def install
-    install_cabal_package
+    system Formula["cabal-install"].bin/"cabal", "v2-update"
+    system Formula["cabal-install"].bin/"cabal", "v2-install", *std_cabal_v2_args
```

Comment:
Is there a situation `cabal` would not be in the superenv PATH or there would be more than one (and thus require the absolute path done here)?

Reply:
Not that I can think of.  I was coding defensively but I'll ease up.